### ISSUE TITLE
The v2.0.1 which support @adonisjs/drive has broken the serialization of the attachment

### ISSUE THREAD

**Issue by Sioood** [batosai/adonis-attachment#1]
### Description
A new version of a package has broken the serialization of the attachment data. The `attachment` field is always serialized as an empty object `{}` even when it contains data.

### Steps to Reproduce
1. Update to the latest version of the package. (v2.0.1)
2. Attempt to serialize an object with attachment data.

### Expected Behavior
The `attachment` field should contain detailed information, for example:

```javascript
 @attachment() 
  declare attachement: Attachment
```

Should return :

```json
{
  "name": "xjfpa4tuxh66p6s3hrgl0pzn.jpg",
  "extname": "jpg",
  "size": 28267,
  "meta": {
    "date": "2010:01:15 18:51:34",
    "host": "Adobe Photoshop CS3 Windows",
    "dimension": {
      "width": 346,
      "height": 346
    },
    "gps": {
      "latitude": 48.862725,
      "longitude": 2.287592,
      "altitude": 2.287592
    },
    "orientation": {
      "value": 1,
      "description": "top-left"
    }
  },
  "mimeType": "image/jpeg",
  "path": "uploads/avatars/xjfpa4tuxh66p6s3hrgl0pzn.jpg",
  "originalName": "aco_bot.jpg",
  "variants": [
    {
      "key": "thumbnail",
      "folder": "uploads/avatars/variants/xjfpa4tuxh66p6s3hrgl0pzn.jpg",
      "name": "ajtugq7224qp9moqyi216vur.webp",
      "extname": "webp",
      "size": 14860,
      "meta": {
        "date": "2010:01:15 18:51:34",
        "host": "Adobe Photoshop CS3 Windows",
        "dimension": {
          "width": 300,
          "height": 300
        },
        "orientation": {
          "value": 1,
          "description": "top-left"
        }
      },
      "mimeType": "image/webp",
      "path": "uploads/avatars/variants/xjfpa4tuxh66p6s3hrgl0pzn.jpg/ajtugq7224qp9moqyi216vur.webp"
    }
  ]
}

**Comment by batosai** [comment]
A fix is in progress.

**Comment by batosai** [comment]
A first beta version has been released (v2.1.1-1). To include URLs, it is necessary to add the `preComputeUrl` option.

```
 @attachment({ preComputeUrl: true }) 
  declare attachement: Attachment
```

**Comment by Sioood** [comment]
I tested it myself, and it's working perfectly fine. Thanks you !
